lib/agenda.js

```diff
@@ -125,13 +123,9 @@ function handleLegacyCreateIndex(err, result, self, cb){
     // Looks like a mongo.version < 2.4.x
     err = null;
     self._collection.ensureIndex(
```

Comment:
Agree to keep it focused. IMO, the `ensureIndex`/`createIndex` should be removed in favor of a simple instruction in the setup in `README.md`